Project.toml

```
@@ -14,7 +14,7 @@ StatsFuns = "4c63d2b9-4356-54db-8cca-17b64c39e42c"
 [compat]
 AbstractMCMC = "2, 3"
 Distributions = "0.23, 0.24, 0.25"
-Libtask = "0.6.7"
+Libtask = "0.7"
```

The compat shouldn't be changed:

```suggestion
Libtask = "0.6.7"
```